minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -128,18 +128,7 @@
     private BufferedReader in;
     private BufferedWriter out;
 
-    public LogWriter(InputStream stream, File logFile) throws IOException {
```

Comment:
Can you remove the entire LogWriter class?  Also can you remove the `logWriters` list?  Or would this cause some problem?